sdks/java/testing/nexmark/build.gradle

```diff
@@ -102,6 +109,39 @@ if (shouldProvideSpark) {
   }
 }
 
+def getNexmarkArgs = {
+  def nexmarkArgsStr =  project.findProperty(nexmarkArgsProperty) ?: ""
+  def nexmarkArgsList = new ArrayList<String>()
+  Collections.addAll(nexmarkArgsList, nexmarkArgsStr.split())
+
+  if (isDataflowRunner) {
+    if (isDataflowRunnerV2) {
+      nexmarkArgsList.add("--experiments=beam_fn_api,use_unified_worker,use_runner_v2,shuffle_mode=appliance")
+      def sdkContainerImage = project.findProperty('sdkContainerImage') ?: project(":runners:google-cloud-dataflow-java").dockerImageName
+      nexmarkArgsList.add("--sdkContainerImage=${sdkContainerImage}")
```

Comment:
Is this line trying to provide images for runner v2? IIRC usually I use two different flags to specify images for runner v2. Are you referring to other v2 setup to use this flag?

Reply:
Flags I have been using with runner v2 is `--experiments=runner_harness_container_image=${image}` and `--worker_harness_container_image=${image}`

Reply:
This is option: https://github.com/apache/beam/blob/018f5e7746b12dc42c77482227e0da429a550248/runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineWorkerPoolOptions.java#L134